Resources/ServerInfo/Guidebook/ServerRules/DefaultRules.xml

```diff
@@ -1,63 +1,38 @@
 <Document>
   # Server Rules
-  This is a roleplay server, meaning that roleplay rules apply. [color=#ff0000]This is also a medium roleplay (MRP) server, meaning that MRP Amendments do apply.[/color]
+The following rules are for the Starlight Space Station 14 servers. Violating these rules may result in a warning, a temporary ban, or a permanent ban. You may also be banned from specific roles/jobs/departments. Bans can be appealed in the official Starlight Discord server.
 
-  Space Station 14 is not like most games. Many rules are designed to require roleplay, and not all rules are intuitive. Please take the time to read and understand the rules before you play so that you aren't surprised. Some of our rules are zero tolerance rules, meaning that a violation will result in an indefinite ban without any warning. Game admins will treat you as if you have read the rules, even if you have not.
+Players are also expected to understand Metashield.
 
-  ## Core Rules
+If you are ever in doubt as to whether or not something is breaking the rules, feel free to contact the admins via Admin Help or ask on the Discord server.
+
+For in-universe laws and rules check Corporate Law and SOP.
+
+Remember, the rules and examples given here are NOT exhaustive. (See Rule 1)
+
+The servers are strictly [bold]16+ only[/bold], unless stated otherwise in the server description that they are +18.
```

Comment:
I would maybe not mention 18+ until we actually roll out a 18+ server, as people will probably ask where it is when it doesnt exist